.github/workflows/reusable-build-test-release.yml

```diff
@@ -1592,6 +1595,183 @@ jobs:
           path: "${{ needs.setup.outputs.directory-path }}/test-results/*.xml"
           reporter: java-junit
 
+  run-escu-tests:
+    if: ${{ needs.test-inventory.outputs.escu == 'true' && ( github.base_ref == 'main' || github.ref_name == 'main' || github.base_ref == 'develop' || github.ref_name == 'develop' ) }}
+    needs:
+      - build
+      - test-inventory
+      - setup
+      - meta
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        splunk: ${{ fromJson(needs.meta.outputs.matrix_latestSplunk) }}
+    container:
+      image: ghcr.io/splunk/workflow-engine-base:2.0.3
```

Comment:
This will need to be updated every time there is a new Docker image. Do we want this behaviour?

Reply:
@artemrys The image we are using (workflow-engine-base) is maintained by us and doesn't get often release.